python/Ganga/new_tests/Unit/TestGangaObject.py

```diff
@@ -296,9 +296,9 @@ def write_read(thread_number):
                     child_num = rand.randint(0, 1000)
                     o.b.a = child_num
                     time.sleep(rand.uniform(0, 1E-6))
-                    self.assertEqual(o.a, num)
```

Comment:
I thought we were moving to make use of the ```assertEqual``` functions? What are we choosing to standardize on?